layers/dot11.go

```diff
@@ -1928,6 +1944,11 @@ func decodeDot11MgmtATIM(data []byte, p gopacket.PacketBuilder) error {
 
 func (m *Dot11MgmtATIM) LayerType() gopacket.LayerType  { return LayerTypeDot11MgmtATIM }
 func (m *Dot11MgmtATIM) CanDecode() gopacket.LayerClass { return LayerTypeDot11MgmtATIM }
+func (m *Dot11MgmtATIM) NextLayerType() gopacket.LayerType {return LayerTypeDot11InformationElement}
```

Comment:
The body of ATIM is null according to the spec - so there should be no next layer

Reply:
Good point, tweaked that.